### DISCUSSION TITLE
CHECKOUT API - Content Security Policy

### DISCUSSION THREAD

**Discussion by FranciscoGarciaHorna** [mercadopago/sdk-js#75] [General]
Cuando implemento bricks y field de checkout API, me sale este error:
![error_field](https://user-images.githubusercontent.com/65722991/177061638-5fb2de4b-098a-4594-8113-7f2700f4ee0e.png)

**Comment by lucmantovani** [comment]
Hola, @FranciscoGarciaHorna!

Me parece que su website tiene la directiva [`frame-src`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/frame-src) del Content Security Policy configurado para cargar iframes solo de BrainTree, Google, Youtube y Vimeo.

Podrías intentar agregar también el host `*.mercadopago.com` para que el browser permita cargar los iframes de fields.
